src/main/kotlin/racingcar/view/InputView.kt

```diff
@@ -0,0 +1,26 @@
+package racingcar.view
+
+import racingcar.value.RacingCarGameInput
+
+class InputView {
```

Comment:
InputView 객체가 별도의 객체를 가지지 않고 Util 함수처럼 쓰인다면 `object`로 변경해보면 어떨까요? 😃 
(ResultView도 동일하게 바꿔볼 수 있을 것 같네요. 😉)

 ```suggestion
object InputView {
```